playbook.yml

```diff
@@ -1,10 +1,14 @@
 - hosts: localhost
   roles:
-  - role: openmicroscopy.omero-web
+  - role: ome.omero_web
   vars:
     ice_version: "3.6"
     ice_install_devel: False
+    ice_install_python: False
     ice_python_wheel: https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
+    omero_web_python3: True
+    omero_web_python_addons:
+    - future
```

Comment:
Why is `future` needed?

Reply:
I copied that directly from the omero-server-docker PR. It was likely added _there_ because I had start this work before everything else was updated. Happy to remove it from both.